prow/config.yaml

```diff
@@ -158,22 +158,23 @@ presubmits:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20180123-1260ba9fc-master
         args:
         - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/kubernetes"
         - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
         - "--root=/go/src"
         - "--service-account=/etc/service-account/service-account.json"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
         - "--timeout=90"
-        env:
-        - name: CADVISOR_APPLICATION_CREDENTIALS
-          value: /etc/cadvisor-service-account/service-account.json
+        - "--clean"
+        - "--git-cache=/root/.cache/git"
+        - "--"
+        - "--node-args=--image-config-file=test/e2e_node/image-config.yaml"
         volumeMounts:
```

Comment:
cache SSD requires adding the host port currently (9999, see other jobs)

Reply:
done